### PR TITLE
Use extend_page instead of deface

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

**Comment by mdellweg** [comment]
Fixes #40

**Comment by timogoebel** [comment]
Don't forget to remove deface from the gemspec.

File: .rubocop.yml [hunk 1]
```diff
@@ -87,3 +88,6 @@ Metrics/PerceivedComplexity:
 
 Style/CaseEquality:
   Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
```
Comment (mdellweg) [inline .rubocop.yml]: What do you all think about this cop?

Comment (sbernhard) [reply .rubocop.yml]: Possible, but you can Enable it and let "rubocop -a" automatically add the frozen string comments.

Comment (timogoebel) [reply .rubocop.yml]: It makes sense to add the comment. Frozen Strings are the default for newer ruby versions. Adding the comment enables this for some older versions.

**Pull request merged**